src/resources/OrganizationAccess/AccessInterfaces.ts

```diff
@@ -22,9 +22,32 @@ export interface AccessParams {
 }
 
 export interface AccessModel {
-    accessLevel: AccessLevel[];
-    callerPartOf: boolean;
+    /*
+     * The access level for this privilege.
+     */
+    accessLevel: AccessLevel;
+    /*
+     * Whether the calling user is part of this group.
+     */
+    callerPartOf?: boolean;
+    /*
+     * The creation date of this privilege.
+     */
+    createdDate?: string;
+    /*
+     * The display name of this privilege.
+     */
     displayName: string;
+    /*
+     * The id of this privilege.
```

Comment:
Same here, its not the id of the privilege, but rather the id of the group in the case of a group holder type and the id of the api key when its an api key.